Examples/SPM/Package.swift

```diff
@@ -4,32 +4,14 @@
 import PackageDescription
 import Darwin.C
 
-// Setup ENV variables when using with Xcode IDE
-//
-//setenv("PACKAGE_URL", "", 1)
-//setenv("PACKAGE_REVISION", "", 1)
-
-func env(_ name: String) -> String? {
-    if let envPointer = getenv(name) {
-        return String(cString: envPointer)
-    } else {
-        print("Can't find environment \(name)")
-        return nil
-    }
-}
-
-guard let packageURL = env("PACKAGE_URL"), let revision = env("PACKAGE_REVISION") else {
-    exit(0)
-}
-
 let package = Package(
     name: "SPMIntegration",
     platforms: [
         .iOS(.v9),
         .macOS(.v10_11)
     ],
     dependencies: [
-        .package(name:"ably-cocoa", url: packageURL, .revision(revision))
+        .package(path: "../../")
```

Comment:
Since when is this possible? Is it works on Xcode 12?

Reply:
Ok it works since 11.4 https://developer.apple.com/documentation/swift_packages/package/dependency/3582846-package nice 🙈